contents/security/passwordStorage.md

```diff
@@ -0,0 +1,150 @@
+<frontmatter>
+  title: Introduction to Password Storage
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Password Storage
+
+**Author: [Jeremy Choo](https://github.com/ChooJeremy)** <br>
+Reviewers: [Amrut Prabhu](https://github.com/amrut-prabhu), [Marvin Chin](https://github.com/marvinchin), [Tan Zhen Yong](https://github.com/Xenonym), [Wang Junming](https://github.com/junming403)
+
+## Overview
+
+Many software applications use a username and password combination as user account credentials for authentication. Obviously, it is not a good idea for the software to store these credentials as <trigger for="pop:plaintext">plain text</trigger> because if someone else were to gain access to them either lawfully (e.g., an employee who has access to the data) or unlawfully (e.g., someone hacking into the data storage), that person can use those credentials directly to impersonate the account owner. This article explains some techniques that can be used to store user credentials more securely:
+
+- Encryption
+- Hashing
+- Salting
+
+<popover id="pop:plaintext" title="_Plaintext_ refers to unencrypted information" placement="top">
+  <div slot="content">
+It is a cryptography term generally referring to text before encyption or after decrypting it. Another term for it is cleartext.
+  </div>
+</popover>
+
+## What is encryption?
+
+Encryption refers to the process of changing a message such that it becomes essentially random text. The opposite process is called _decryption_, which converts the text back into the original message. In order to do this, it takes in an <trigger for="pop:encrypt-key">encryption key</trigger>, which is then re-used in the decryption process. 
```

Comment:
>Encryption refers to the process of changing a message such that it becomes essentially random text.

I have concerns about this definition. What `message`? Is converting plain text to random text is encryption if there is no way to recover the original text?